server/api/tests/test_models.py

```diff
@@ -1,3 +1,88 @@
-from django.test import TestCase
+from contextlib import suppress
+from django.core.exceptions import ValidationError
+
+
+from api.models import (UserCategoryHistory, UserEventHistory)
+from api.tests.base_test_setup import BaseSetup
+
+
+class UserEventHistoryTests(BaseSetup):
+
+    def test_user_create_event_history(self):
+
+        with suppress(ValidationError):
+            new_user_event_history = UserEventHistory.objects.create(
```

Comment:
I am of the assumption that the UserEventHistory is created the moment an Event is created by a user. So I feel this test should just check if the event has been registered to the user's history. @tosmak16 Your thoughts

Reply:
Thanks @makindemayowa That will be done later. Because we intend to use Django signals to implement that scenerio. The test will be written for such scenerio.